editor/src/uuiui/styles/theme/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Theme } from '../../../components/editor/store/editor-state'
+import { useEditorState } from '../../../components/editor/store/store-hook'
+import { dark } from './dark'
+import { light } from './light'
+import { ColorTheme, colorTheme, darkColorTheme, UtopiaTheme } from './utopia-theme'
+import type { UtopiColor } from '../utopi-color-helpers'
+
+export const colorThemeCssVariables = Object.keys(light).reduce((prev, curr) => {
```

Comment:
I think what you want to use here is `mapValues` from https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/shared/object-utils.ts#L92-L107